test/unit/test-storage.js

```diff
@@ -313,6 +313,49 @@ describes.sandboxed('Storage', {}, (env) => {
           });
       });
 
+      it('should get unexpired value based on duration', async () => {
+        const dateStub = env.sandbox.stub(Date, 'now');
```

Comment:
Do you think we can use `lolex` or `useFakeTimers` to avoid directly mocking Date.now?